docs/sql-ref-syntax-qry-select.md

```diff
@@ -18,8 +18,132 @@ license: |
+  <dd>
+    Specifies a source of input for the query. It can be one of the following.
+    <ol>
+      <li>Table relation</li>
+      <li>Join relation</li>
+      <li>Table valued function</li>
+      <li>Inlined table</li>
+      <li>Subquery</li>    
+    </ol>
+  </dd>
+  <dt><code><em>WHERE</em></code></dt>
+  <dd>
+    Filters the result of the FROM clause based on the supplied predicates.
+  </dd>
+  <dt><code><em>GROUP BY</em></code></dt>
+  <dd>
+    Specifies the expressions that are used to group the rows. This is used in conjunction with aggregate functions
+    (MIN, MAX, COUNT, SUM, AVG) to group rows bsed on the grouping expressions.
+  </dd>
+  <dt><code><em>HAVING</em></code></dt>
+  <dd>
+    Specifies the predicates by which the rows produced by GROUP BY are filtered. The HAVING clause is used to
+    filter rows after the grouping is performed
+  </dd>
+  <dt><code><em>ORDER BY</em></code></dt>
+  <dd>
+    Specifies an ordering of the rows of the complete result set of the query. The output rows are ordered
+    across the partitions. This parameter is mutually exclusive with <code>SORT BY</code>, 
+    <code>CLUSTER BY</code>, or <code>DISTRIBUTE BY</code> and can not be specified together.
+  </dd>
+  <dt><code><em>SORT BY</em></code></dt>
+  <dd>
+    Specifies an ordering by which the rows are ordered within each partition. This parameter is mutually
+    exclusive with <code>ORDER BY</code>, <code>CLUSTER BY</code> and can not be specified together.
+  </dd>
+  <dt><code><em>CLUSTER BY</em></code></dt>
+  <dd>
+    Specifies a set of expressions that is used to repartition and sort the rows. Using this clause has
+    the same effect of using <code>DISTRIBUTE BY</code> and <code>SORT BY</code> together. 
+  </dd>
+  <dt><code><em>DISTRIBUTE BY</em></code></dt>
+  <dd>
+    Specifies a set of expressions by which the result rows are repartitioned. This parameter is mutually 
+    exclusive with <code>ORDER BY</code> and <code>CLUSTER BY</code> and can not be specified together. 
+  </dd>
+  <dt><code><em>LIMIT</em></code></dt>
+  <dd>
+    Specifies the maximum number of rows that can be returned by a statement or subquery. This clause 
+    is mostly used in the conjunction with <code>ORDER BY</code> to produce deterministic result. 
+  </dd>
+  <dt><code><em>boolean_expression</em></code></dt>
+  <dd>
+    Specifies a expression with a return type of boolean.
+  </dd>
+  <dt><code><em>expression</em></code></dt>
+  <dd>
+    Specifies a combination of one or more values, operators, and SQL functions that evaluates to a value.
+  </dd>
+  <dt><code><em>named_window</em></code></dt>
+  <dd>
+    Specifies aliases for one or more source window specifications. The source window specifications can 
+    be referenced in the widow definitions in the query.
+  </dd>
+</dl>
+
+### Related clauses
+- [FROM clause](sql-ref-syntax-qry-select-from.html)
+- [WHERE clause](sql-ref-syntax-qry-select-where.html)
```

Comment:
@srowen Are you okay to keep these or we want to remove the related clauses section add add as we go ? Please let me know.